frontend/src/assets/css/styles.scss

```
@@ -1,6 +1,9 @@
 // noinspection CssInvalidAtRule
 @import-normalize;
 
+@import "./variables";
+@import "https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,700&display=swap";
```

Imports from 3rd parties should be located before the custom styles/code. It would be better to have the google fonts import right after the `~normalize`.

```suggestion
@import-normalize;
@import "https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,700&display=swap";

@import "./variables";
```